### PR TITLE
docs: correction on weak subjectivity

### PR DESCRIPTION
**Pull request by eccentricexit**
**Motivation**

There is incorrect information on the documentation. Weak subjectivity is not a path to execute long range attacs.

**Description**

This is just a really small documentation correction.

**Steps to test or reproduce**

### CODE REVIEW

File: docs/usage/beacon-management.md [hunk 1]
```diff
@@ -101,8 +101,8 @@ A young testnet should take a few hours to sync. If you see multiple or consiste
 
 ### Checkpoint Sync
 
-If you are starting your node from a blank db/genesis (or from last saved state in db) in a network which is now far ahead, your node is susceptible to "long range attacks" via something called weak subjectivity.
-[Read Vitalik's illuminating post on the same](https://blog.ethereum.org/2014/11/25/proof-stake-learned-love-weak-subjectivity/).
+If you are starting your node from a blank db/genesis (or from last saved state in db) in a network which is now far ahead, your would be susceptible to "long range attacks". Ethereum's solution to this is something called weak subjectivity.
```
Comment (philknows) [inline docs/usage/beacon-management.md:104]:
```suggestion
If you are starting your node from a blank db/genesis (or from last saved state in db) in a network which is now far ahead, your node would be susceptible to "long range attacks". Ethereum's solution to this is something called weak subjectivity.
```

**Comment by philknows** [comment]
Thanks for the correction. Please sign the CLA and we'll push it through.

**Comment by wemeetagain** [comment]
bump

**Comment by eccentricexit** [comment]
Unfortunately I do not have time to study the CLA. Feel free to close the issue and correct it with another account.

**Comment by matthewkeil** [comment]
Moved to #5712 and closing this PR

**Pull request closed without merging**